pipelines/ProcessCcd.yaml

```diff
@@ -1,7 +1,13 @@
 description: ProcessCcd - A set of tasks to run when processing raw images.
 tasks:
   isr: lsst.ip.isr.IsrTask
-  characterizeImage: lsst.pipe.tasks.characterizeImage.CharacterizeImageTask
+  characterizeImage:
+    class: lsst.pipe.tasks.characterizeImage.CharacterizeImageTask
+    # We want a larger psf kernel, to include the calibFlux 12 pixel radius.
+    config:
+      python: |
+        config.measurePsf.makePsfCandidates.kernelSize = 25
+        config.measurePsf.psfDeterminer["piff"].kernelSize = 25
```

Comment:
I think you'd also need to add this to the instrument-specific pipelines after all, because they have a `characterizeImage.py` with the config-reset hack. 😞

Reply:
Hmm... no, it looks like it gets propagated. I think that means I was wrong about the evaluation order. 😕

Reply:
Jim Bosch was surprised when I asked about this behavior and suggested it could be a bug, so I don't think we can count on the current behavior in the future.